src/Symfony/Component/Console/Descriptor/DescriptorInterface.php

```diff
@@ -21,7 +21,8 @@
 interface DescriptorInterface
 {
     /**
-     * Describes an object if supported.
+     * Describes an object if the class that
+     * the object is an instance of, is supported.
```

Comment:
the comma should be removed and the description put on a single line to me.

Reply:
Hi @nicolas-grekas, thank you for reviewing my Pull Request. I have pushed up a further commit for this update. Please can you review whenever you have a chance to do so?